### ISSUE TITLE
Can't load pivot table with borders

### ISSUE THREAD

**Issue by cwuethrich** [ClosedXML/ClosedXML#1511]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature
- [ ] Question

**Did you test against the latest CI build?**
- [x] Yes
- [ ] No

**Version of ClosedXML**

0.95.3

**What is the current behavior?**

Can't load a pivot table with borders. Already `new XLWorkbook("PivotWithBorder.xlsx")` fails.

**What is the expected behavior or new feature?**

Also a pivot table with borders can be loaded with a `Workbook`.

**Is this a regression from the previous version?**

It worked with 0.94.2, Doesn't work since 0.95.0. 

## Reproducibility

**Code to reproduce problem:**
```c#
public void Main()
{
    var wb = new XLWorkbook(@"C:\temp\PivotWithBorder.xlsx");
}
```

- [X] I attached a sample spreadsheet.  (You can drag files on to this issue)
[PivotWithBorder.xlsx](https://github.com/ClosedXML/ClosedXML/files/5171168/PivotWithBorder.xlsx)

**Comment by SfWt** [comment]
I can reproduce the failure with the provided file in my app using ClosedXml 0.95.3 also.

From my investigation the value of the "_container" element of the _ClosedXML.Excel.XLBorder_ class is null and therefore calls to the _Set..._ methods are failing with a NullReferenceException. I did the following change to the XLBorder constructor and the file was loaded successfully. But because I'm not experienced enough with the CloseXML code I'm not sure if this is just a workaround.

``` C#
public XLBorder(IXLStylized container, XLStyle style, XLBorderValue value)
{
    //_container = container;
    _container = container ?? new XLStylizedEmpty(style ?? XLStyle.Default);
    _style = style ?? _container.Style as XLStyle ?? XLStyle.CreateEmptyStyle();
    _value = value;
}
```

Maybe this can be of any help in fixing this issue.

**Comment by igitur** [comment]
Pivot tables are still work in progress and pivot table formatting is even more experimental.

**Comment by cwuethrich** [comment]
> Pivot tables are still work in progress and pivot table formatting is even more experimental.

I don't really use pivot tables. But o have to load a workbook with a pivot table, because I have to load data from another worksheet. So with this exception it isn't possible to load the workbook.

**Comment by jahav** [comment]
Resolved in #1928
